vnext/ReactWindowsCore/Utils.cpp

```diff
@@ -4,35 +4,31 @@
 #include "Utils.h"
 #include <regex>
 
-using namespace std;
+using std::string;
 
 namespace Microsoft::React {
 
 Url::Url(const string &source) {
-  //      ( 1 )              ( 2 )   ( 3 (4) )   ( 5 )    ( 6 (7) )
-  regex expression("(http|https|ws|wss)://([^:/\\?]+)(:(\\d+))?(/[^\\?]*)?(\\?(.*))?$");
-  //     protocol             host       port     path        query
-  cmatch match;
-  int index = 0;
-  if (regex_match(source.c_str(), match, expression)) {
-    ++index;
-    this->scheme = string(match[index].first, match[index].second);
+  //                           ( 1 )              ( 2 )   ( 3 (4) )   ( 5 )    ( 6 (7) )
+  std::regex expression("(http|https|ws|wss)://([^:/\\?]+)(:(\\d+))?(/[^\\?]*)?(\\?(.*))?$");
```

Comment:
`std::regex` is surprisingly slow. Not needed for this check-in, but it would be nice if we could avoid having our own custom URL parsing logic. E.g. I'm sure CppWinRT has one we could use.

Reply:
Track that feedback with an issue then?